erpnext/buying/doctype/purchase_order/test_purchase_order.py

```diff
@@ -632,7 +632,7 @@ def test_po_for_blocked_supplier_payments_past_date(self):
 			else:
 				raise Exception
 
-	def test_terms_does_not_copy(self):
+	def test_terms_are_not_copied_if_automatically_fetch_payment_terms_is_unchecked(self):
```

Comment:
The order of test case execution is not fixed. There may be cases when another test case that is enabling the setting may get run first hence this test is failing. Disable the setting agiain in other test case once the test is complete

Reply:
Done.